tests/openvino/test_modeling.py

```diff
@@ -943,7 +943,7 @@ def test_beam_search(self, model_arch):
             eos_token_id=None,
         )
 
-        if model_arch == "minicpm":
+        if model_arch in ["minicpm", "internlm2"]:
```

Comment:
is this okay ? idk why the generation loop is not deterministic with `internlm2`

Reply:
yes, it is due to random initialization of pt model